### PR TITLE
Change design feedback

### PR DESCRIPTION
**Pull request by piya43044**
**Checklist**
- [x] Code adheres to **[Git and Pull Request Guidelines]**
- [x] Code adheres to **[Angular Guidelines]**
- [x] Code adheres to **[Front-end Checklist]**

### CODE REVIEW

**Comment by AshaPatel216** [comment]
Pull request title must be same as branch name with the first latter capital
change-design-feedback -> Change design feedback

File: src/app/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+              <tr>
+                <td colspan="3">
+                    <div class="dashboard-header d-flex justify-content-between align-items-center">
+                        <div>
+                            <h4>Dashboard</h4>
+                        </div>
+        
+                        <!-- Add expense button and settle up button -->
+                        <div class="expense-settleup">
+                            <button class="btn btn-danger" routerLink="/expense/expense-add">Add Expense</button>
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.html]: Missing button type

File: src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
+                            <button class="btn btn-success" routerLink="/settleUp">Settle Up</button>
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.html]: Same as above

File: src/app/dashboard/dashboard.component.html [hunk 3]
```diff
@@ -41,20 +42,27 @@ <h4>Dashboard</h4>
...
+    
+                
+                <!-- Owe/Owed Details -->
+                <tr *ngFor="let detail of balanceDetail">
+                    <td colspan="3">
+                        <div *ngIf="detail.owe =='owe'" class="d-flex flex-column justify-content-center align-items-center">
+                            <a routerLink="/friends/friends-list/{{detail.person}}">{{detail.person}}</a>
+                            <p>You owe {{detail.currency}}{{detail.amount}}</p>
+                        </div>
+                        <div *ngIf="detail.owe == 'owed'" class="d-flex flex-column justify-content-center align-items-center">
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.html]: Use strong type for comparision

File: src/app/dashboard/dashboard.component.html [hunk 4]
```diff
@@ -41,20 +42,27 @@ <h4>Dashboard</h4>
...
+            </thead>
+      
+            <!-- Dashboard Detail -->
+            <tbody>
+    
+                
+                <!-- Owe/Owed Details -->
+                <tr *ngFor="let detail of balanceDetail">
+                    <td colspan="3">
+                        <div *ngIf="detail.owe =='owe'" class="d-flex flex-column justify-content-center align-items-center">
```
Comment (AshaPatel216) [inline src/app/dashboard/dashboard.component.html]: Use strong type for comparison

File: src/app/expense/expense-list/expense-list.component.scss [hunk 5]
```diff
@@ -10,14 +10,31 @@
...
     }
 
+    /* Action button of edit / delete button */
+    .btn-action {
+        padding: 5px;
+        border: none;
+        
+        /* Edit / delete icon */
+        i {
+            font-size: 20px;
```
Comment (AshaPatel216) [inline src/app/expense/expense-list/expense-list.component.scss]: use proper font variables. make list of font size and create its variable. The maximum font size list should be 6 to 7 and it to whole website. Please make correction to overall project

File: src/app/expense/expense-list/expense-list.component.ts [hunk 6]
```diff
@@ -13,4 +14,17 @@ export class ExpenseListComponent {
     { expenseName:'Notes', groupName:'Vanaras' , paidBy:'Nikita', lentOrBorrow:'borrow'},
     { expenseName:'Taxi', groupName:'Goa' , paidBy:'Harish', lentOrBorrow:'lent'},
   ]
+
+  // Constructor
+  constructor( private router: Router){};
+
+  // Navigate to edit form
+  navigateToEditForm(index: number): void{
```
Comment (AshaPatel216) [inline src/app/expense/expense-list/expense-list.component.ts:25]: Use comment formatting
image

with @params, @return if it is there
make correction to overall code for practice

File: src/app/friends/friends-list/friends-list.component.html [hunk 7]
```diff
@@ -3,75 +3,118 @@
...
-                </div>
+                <td colspan="4">
+                  <div class="d-flex justify-content-between align-items-center">
+
+                    <!-- Back icon -->
+                    <a routerLink="/dashboard"><i class="fa fa-arrow-circle-left"></i></a>
+
+                    <!-- Friend heading -->
+                    <h4>Friends</h4>
+                    <button class="btn btn-success" routerLink="/friends/friends-add">+ Add Friend</button>
```
Comment (AshaPatel216) [inline src/app/friends/friends-list/friends-list.component.html]: Missing button type

File: src/app/sidebar/sidebar.component.ts [hunk 8]
```diff
@@ -7,4 +9,19 @@ import { Component, Input } from '@angular/core';
 })
 export class SidebarComponent {
   @Input() sideNavStatus: Boolean = false;
+  currentUrl: string = '/dashboard';
+
+  // Constructor
+  constructor( private router: Router){};
```
Comment (AshaPatel216) [inline src/app/sidebar/sidebar.component.ts:15]: Add one space before oninit function for readability

File: src/app/expense/expense-list/expense-list.component.ts (hunk 6 above)
Comment (piya43044) [reply src/app/expense/expense-list/expense-list.component.ts:25]: done

File: src/app/sidebar/sidebar.component.ts (hunk 8 above)
Comment (piya43044) [reply src/app/sidebar/sidebar.component.ts:15]: done

File: src/app/friends/friends-list/friends-list.component.html (hunk 7 above)
Comment (piya43044) [reply src/app/friends/friends-list/friends-list.component.html]: done

File: src/app/expense/expense-list/expense-list.component.scss (hunk 5 above)
Comment (piya43044) [reply src/app/expense/expense-list/expense-list.component.scss]: done

File: src/app/dashboard/dashboard.component.html (hunk 4 above)
Comment (piya43044) [reply src/app/dashboard/dashboard.component.html]: done

File: src/app/dashboard/dashboard.component.html (hunk 3 above)
Comment (piya43044) [reply src/app/dashboard/dashboard.component.html]: done

File: src/app/dashboard/dashboard.component.html (hunk 2 above)
Comment (piya43044) [reply src/app/dashboard/dashboard.component.html]: done

File: src/app/dashboard/dashboard.component.html (hunk 1 above)
Comment (piya43044) [reply src/app/dashboard/dashboard.component.html]: done

**Review by AshaPatel216 — APPROVED** [review]

**Pull request merged**